models/model.js

```diff
@@ -0,0 +1,31 @@
+const db = require("../db/connection");
+
+exports.selectTopics = () => {
+
+    sqlQuery = 'SELECT * FROM topics';
+
+    return db        
+        .query(sqlQuery)
+        .then((data) => {
+            const topics = data.rows;
+            return topics;
+        })
+};
+
+exports.selectArticleById = (article_id) => {
+
+    sqlQuery = `SELECT u.username, a.title, a.article_id, a.body, a.topic, a.created_at, a.votes
+                FROM users u, articles a
+                WHERE u.username = a.author
+                AND a.article_id = $1;`
+
```

Comment:
The seed should be giving you access to author on the article table so I don't think you need to do this. However it does work! Change it if you like or if you prefer this way, fine by me!